### PR TITLE
Feature/ Adding find gift.

### PR DESCRIPTION
**Pull request by Mekhak11**
https://github.com/user-attachments/assets/f60941ea-d05d-4bdf-927d-75781a569951

### CODE REVIEW

File: Birthday/Birthday/Modules/Tabbar/View/TabBarView.swift [hunk 1]
```diff
@@ -125,11 +127,14 @@ extension TabBarView {
...
-    case details(viewModel: BirthdayDetailsViewModel, birthday: BirthdayModel)
+    case details(viewModel: BirthdayDetailsViewModel)
+    case shops(viewModel: ShopViewModel)
     
     var id: UUID {
       switch self {
-      case let .details(viewModel, _): viewModel.id
+      case let .details(viewModel): viewModel.id
+      case let .shops(viewModel): viewModel.id
+
```
Comment (AkselAvetisyan) [inline Birthday/Birthday/Modules/Tabbar/View/TabBarView.swift]: additional space

File: Birthday/Birthday/Modules/Tabbar/View/TabBarView.swift [hunk 2]
```diff
@@ -139,8 +144,11 @@ extension TabBarView {
     
     func hash(into hasher: inout Hasher) {
       switch self {
-      case let .details(viewModel, _):
+      case let .details(viewModel):
```
Comment (AkselAvetisyan) [inline Birthday/Birthday/Modules/Tabbar/View/TabBarView.swift]: There’s no need for a switch case here.

File: Birthday/Birthday/Modules/Tabbar/View/TabBarView.swift (hunk 1 above)
Comment (Mekhak11) [reply Birthday/Birthday/Modules/Tabbar/View/TabBarView.swift]: fixed.

File: Birthday/Birthday/Modules/Tabbar/View/TabBarView.swift (hunk 2 above)
Comment (Mekhak11) [reply Birthday/Birthday/Modules/Tabbar/View/TabBarView.swift]: fixed.

**Review by AkselAvetisyan — APPROVED** [review]

**Pull request merged**